datameta/api/users.py

```diff
@@ -35,7 +35,7 @@ class UserUpdateRequest(DataHolderBase):
 class UserResponseElement(DataHolderBase):
     """Class for User Update Request communication to OpenApi"""
     id: dict
-    name: str
+    name: str # why is this name when it is called fullname in the db?
```

Comment:
Please change it as you like, but please make sure to change it in all other requests/responses of all other endpoints dealing with the username, too. This also needs changes in the openapi.yaml.

Reply:
Should it be `name` or `fullname` then? I'd assume changing to `name` is easier.

Reply:
Fine by me, I just wanted to flag this, hence the comment.